doc/bdDijkstra/bdDijkstra-family.rst

```diff
@@ -12,8 +12,8 @@
 .. rubric:: Previous versions of this page
 
 * **Supported versions:**
-  `Latest <https://docs.pgrouting.org/latest/en/pgrbdDijkstra.html>`__
-  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_bdDijkstra.html>`__)
+  `Latest <https://docs.pgrouting.org/latest/en/pgrbdDijkstra-family.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_bdDijkstra-family.html>`__)
```

Comment:
Line 15 y 16 remove the `pgr_` from `pgr_bdDijkstra-family.html`

Reply:
The file name is `bdDijkstra-family.rst` So after the `/en/` it should read `bdDijkstra-family.html`